docker-compose.yml

```diff
@@ -24,7 +24,17 @@ services:
     - POSTGRES_DB=${DB_NAME:-nodefactory}
     ports:
     - ${DB_LOCAL_PORT:-9999}:5432
+  swagger:
+    image: swaggerapi/swagger-ui
+    container_name: "swagger_docs"
+    environment:
+      BASE_URL: /api/swagger
+      SWAGGER_JSON: /swagger.yaml
+    volumes:
+      - "./swagger.yaml:/swagger.yaml"
+    ports:
+      - "8080:8080"
```

Comment:
8080 on the left side should be configurable using ENV variable (check above for database)